tests/wicked/ipv6/sut/t01_managed_off_other_off_pl_ne64.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: IPv6 - Managed off, other off, prefix length != 64
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base wicked::IPv6;
+use strict;
+use warnings;
+use lockapi;
+
+sub run {
+    my ($self, $ctx) = @_;
+    mutex_wait('radvdipv6t01');
+    my $iface = $ctx->iface();
+    $self->setup_interface($iface, 'static');
+    $self->wait_for_router_announcement($iface);
+    $self->retrigger_dhcp_requests($iface);
+    $self->obtain_address_by_dhcpv4($iface, 0);
+    $self->obtain_autonomous_ipv6_address($iface, 0);
+    $self->obtain_address_by_DHCPv6($iface, 0);
+    $self->address_should_be_tentative($iface, 0);
+    $self->obtain_default_ipv6_route($iface, 1);
+    $self->obtain_default_dns_server_by_dhcpv4(0);
+    $self->obtain_default_dns_server_by_dhcpv4(0);
```

Comment:
why you call it second time ?

Reply:
should be `obtain_default_dns_server_by_dhcpv6` I will update it once I get more comments